src/withError.ts

```diff
@@ -1,18 +1,23 @@
-import { withFormClass } from './withFormClass'
+import { Constructor, CustomElement } from './types'
+
 import { EVENTS } from './withForm'
-import get from 'lodash.get'
+import get from 'lodash-es/get'
+import { withFormClass } from './withFormClass'
 
 // Super need to have "name" attribute
 // HoC which returns "error" and "touched" from form by "name"
-export const withError = Сomponent => {
-  return withFormClass(
```

Comment:
Note the change here:

```typescript
// from
return withFormClass(class extends Component {...});

// to
return class extends withFormClass(Component) {...}
```
I believe those are equivalent but the update is easier to read ?